database/migrations/sql/000004_seed_node_attribute.up.sql

```diff
@@ -53,7 +53,17 @@ values
         '{{CORE_PLUGIN_ID}}',
         'world_template',
         '{
-          "objects": [],
+          "objects": [{"object_id": "b3dbfce9-c635-4506-a823-09954a28dcd1",
```

Comment:
we do need a separate db migration to update the world_template node_attribute  on all our deploy environments